### DISCUSSION TITLE
How to get trough the detector

### DISCUSSION THREAD

**Discussion by dudongboxian** [kaliiiiiiiiii/Selenium-Profiles#70] [Q&A] [answered]
I mean how to pass https://hmaker.github.io/selenium-detector/ with the [/dev_stuff/devtools_anti_detector.js](https://github.com/kaliiiiiiiiii/Selenium-Profiles/blob/master/dev_stuff/devtools_anti_detector.js)

**Comment by ttlns** [comment, marked as answer]
That script was for some testing. I'm pretty sure it would be detectable.

`driver.execute_cdp_cmd("Runtime.disable",{})` might help you, but doesn't completely make it pass (and causes some issues)
```python
from selenium_profiles.webdriver import Chrome
from selenium_profiles.profiles import profiles
from selenium_injector.types.by import By  # locate elements

profile = profiles.Windows()

driver = Chrome(profile, options=options, uc_driver=True,injector_options={"debug":True})
driver.execute_cdp_cmd("Runtime.disable",{})
injector = driver.profiles.injector

# get url
driver.get('https://hmaker.github.io/selenium-detector/')

driver.quit()  # Execute on the End!
```

See https://github.com/kaliiiiiiiiii/Selenium-Profiles/issues/57 for more information

**Comment by ttlns** [comment]
Closing as it seems to be resolved
